docs/documentation/native/native2.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Native Automation 2.0 (Native2)
+---
+
+<Info>
+Native2 is available starting from Patrol version `3.6.0`.
+</Info>
+
+## What is `native2`?
+
+Native2 was created to address fundamental limitations in the original native automation approach. The original native API was primarily designed for Android, and attempts to make a single `Selector` work across both platforms proved problematic because:
+
+- **Platform-specific identifiers**: iOS and Android use different identifier systems (Android's `resourceName` vs iOS's `identifier`) and a single selector approach couldn't effectively handle the fundamental differences between iOS and Android element identification
```

Comment:
remove list, make it just a part of a sentence, maybe bolded. It looks weird when it's only one point

Reply:
Also the wording here is a bit strange - what's an identifier system? Did you mean selectors arguments? It looks here like only "Android's `resourceName` vs iOS's `identifier`" is different. I would ditch the word "identifier" and use selector instead - it's quite universal name used in many frameworks and doesn't indicate our `Selector`